interfaces/system_key_test.go

```diff
@@ -389,13 +503,13 @@ func (s *systemKeySuite) TestSystemKeysUnmarshalSame(c *C) {
 			"trap",
 			"user_notif"
 		],
-		"version": 10
-	}`
+		"version": 12
+	}`, strconv.FormatBool(promptingSupported), strconv.FormatBool(promptingSupportedAndEnabled))
```

Comment:
Then this can just be
```suggestion
	}`, promptingSupported, promptingSupportedAndEnabled)
```